features/json-ld/context.feature

```diff
@@ -8,26 +8,16 @@ Feature: JSON-LD contexts generation
     Then the response status code should be 200
     And the response should be in JSON
     And the header "Content-Type" should be equal to "application/ld+json"
-    And the JSON should be equal to:
+    And the JSON should be valid according to this schema:
     """
     {
-      "@context": "/contexts/Entrypoint",
-      "@id": "/",
-      "@type": "Entrypoint",
-      "abstractDummy": "/abstract_dummies",
```

Comment:
What about a custom assertion `Then :json, should have every resource listed`?
Do we hard code values or should we take the `ResourceNameCollection` into consideration to avoid having the same file modified across PR's?